typings/pg-sql2.d.ts

```diff
@@ -6,6 +6,6 @@ declare module 'pg-sql2' {
 
   export function compile(query: SQLQuery): QueryConfig
   export function query(strings: TemplateStringsArray, ...values: Array<SQLNode | SQLQuery>): SQLQuery
-  export function value(val: mixed): SQLNode
+  export function value(val: any): SQLNode
```

Comment:
Why was this change required? Specifically this should only receive JSON-like values, I think...

Reply:
i confess.  laziness as i wasn't not sure how to import a locally defined interface :/ `../src/interfaces`